constructorio-client/src/main/java/io/constructor/client/models/ResultGroup.java

```diff
@@ -14,6 +15,9 @@ public class ResultGroup {
     @SerializedName("path")
     private String path;
 
+    @SerializedName("path_list")
+    private List<ResultGroupPathListItem> pathList;
```

Comment:
wyt about just naming this PathListItem or PathListGroup.. Something like that.

No harm, it's fine as is but just thought it was long 😅